### DISCUSSION TITLE
"Show Tabs List" support non-ASCII please.

### DISCUSSION THREAD

**Discussion by gwankyun** [doublecmd/doublecmd#1784] [Suggestions]
DC can't foucs tab which tab name has non-ASCII char.

Double Commander
Version: 1.1.17 gamma
Revision: 548
Commit: f4189f213
Build date: 2024/08/19
Lazarus: 3.5.0.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 11 23H2 x86_64

---

Double Commander
Version: 1.1.18 gamma
Revision: 568
Commit: c45365d1b
Build date: 2024/09/21
Lazarus: 3.5.0.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 11 23H2 x86_64

**Fixed** when tab is normal state, but locked tab can not foucs.

"f文件" can focus, "*f文件" can not. 

Type '*‘ can focus it.

**Comment by alexx2000** [comment]
Copy here Double Commander version (Menu->Help->About->Copy to clipboard).

**Reply by gwankyun** [reply]
OK

**Comment by alexx2000** [comment]
#1890
